### PR TITLE
Update registry keys for AdoptOpenJDK

### PR DESCRIPTION
**Pull request by golubev-ivan**
Added registry keys for AdoptOpenJDK version greater or equal to 11.0.13+7 that is distributed with hotspot JVM:
- "Software\Eclipse Adoptium\JDK"
- "Software\Eclipse Adoptium\JRE"

Related issue: #3777

Tested detection of JDK\JRE in the next configurations:
- AdoptOpenJDK JDK of 11.0.12+7version with openj9
- AdoptOpenJDK JRE of 11.0.12+7 version with openj9
- AdoptOpenJDK JDK of 11.0.12+7 version with hotspot
- AdoptOpenJDK JRE of 11.0.12+7 version with hotspot
- AdoptOpenJDK JDK of 11.0.13+8version with openj9
- AdoptOpenJDK JRE of 11.0.13+8 version with openj9
- AdoptOpenJDK JDK of 11.0.13+8 version with hotspot
- AdoptOpenJDK JRE of 11.0.13+8 version with hotspot
- AdoptOpenJDK JDK of 11.0.14+1version with openj9
- AdoptOpenJDK JRE of 11.0.14+1 version with openj9
- AdoptOpenJDK JDK of 11.0.14+1 version with hotspot
- AdoptOpenJDK JRE of 11.0.14+1 version with hotspot

### CODE REVIEW

**Comment by anatolybolshakov** [comment]
@golubev-ivan could you please share test results also?

File: src/Misc/layoutbin/powershell/Add-JavaCapabilities.ps1 [hunk 1]
```diff
@@ -58,8 +62,9 @@ if (-not $latestJre) {
...
+    $null = Add-CapabilityFromRegistryWithLastVersionAvailable -PrefixName 'java_' -PostfixName '_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jdk9AndGreaterNameAdoptOpenJREEclipseAdoptium -ValueName 'Path' -Value ([ref]$latestJre) -VersionSubdirectory $jvmHotSpot -MinimumMajorVersion $minimumMajorVersion9
+    $null = Add-CapabilityFromRegistryWithLastVersionAvailable -PrefixName 'java_' -PostfixName '_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jdk9AndGreaterNameAdoptOpenJREEclipseFoundation -ValueName 'Path' -Value ([ref]$latestJre) -VersionSubdirectory $jvmHotSpot -MinimumMajorVersion $minimumMajorVersion9
+	$null = Add-CapabilityFromRegistryWithLastVersionAvailable -PrefixName 'java_' -PostfixName '_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jdk9AndGreaterNameAdoptOpenJRESemeru -ValueName 'Path' -Value ([ref]$latestJre) -VersionSubdirectory $jvmOpenj9 -MinimumMajorVersion $minimumMajorVersion9
```
Comment (anatolybolshakov) [inline src/Misc/layoutbin/powershell/Add-JavaCapabilities.ps1:67]: Seems like redundant indent here

**Review by EzzhevNikita — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!

**Comment by golubev-ivan** [comment]
Hi @mmrazik, could you please take a look at this PR?

**Pull request merged**